src/main/java/org/example/FiltersPage.java

```diff
@@ -39,6 +39,10 @@ public void clearMinPriceField () {
         minPriceField.clear();
     }
 
+    public WebElement getMinPriceField() {
```

Comment:
Remove this method. You shouldn't be able to "work" with the `minPriceField` element directly from a class with tests